### PR TITLE
Torreanalysis

### PR DESCRIPTION
**Pull request by angelaxliu**
Node colors working

### CODE REVIEW

File: de/figures/figure4.py [hunk 1]
```diff
@@ -13,3 +13,5 @@ def makeFigure():
     subplotLabel(ax)
 
     return f
+
+
```
Comment (Farnazmdi) [inline de/figures/figure4.py:17]: You should only have 1 white line after the end of the last function in a file. (This is a linting problem)
Please remove one of the lines.

File: de/figures/torre [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import numpy as np
+from ..importData import importRNAseqKO()
+from ..importData import formMatrix()
+
+
+
```
Comment (Farnazmdi) [inline de/figures/torre:6]: Do you need this file? ("torre.py")
If not, please delete the file from the figures folder.

**Review by Farnazmdi — COMMENTED** [review]
Everything else looks good!
You can ask Dr. Meyer to review and merge this PR as it is, and work on completing the list of genes in another branch, 
or wait and add the list of genes in this PR.

**Comment by angelaxliu** [comment]
@aarmey The nodes are color coded, but it only works for the genes shown in Torre's Figure 4D, so I'm working on matching our list against their metadata.

**Comment by Farnazmdi** [comment]
@angelaxliu please do `git pull` to update your local.

**Pull request closed without merging**